tests/integration-tests/tests/common/utils.py

```diff
@@ -231,3 +235,33 @@ def get_route_tables(subnet_id, region):
         Filters=[{"Name": "association.subnet-id", "Values": [subnet_id]}]
     )
     return [table["RouteTableId"] for table in response["RouteTables"]]
+
+
+def gather_system_report(cluster, request, scheduler_commands, remote_command_executor):
+    out_dir = request.config.getoption("output_dir")
+    logging.info(f"Get system information about test: {request.node.name}")
+    logging.info(f"Run command on head node to get system information about test: {request.node.name}")
+    result = remote_command_executor.run_remote_script(SYSTEM_REPORT_SCRIPT, args=["/tmp"])
+    logging.debug(f"result.failed={result.failed}")
+    logging.debug(f"result.stdout={result.stdout}")
+    logging.info(f"Copy results from remote cluster into: {out_dir}/system_report/systemreport_compute_node_{request.node.name}.tar.gz")
```

Comment:
systemreport_compute_node --> should be systemreport_head_node

Reply:
Right. Done